library_generation/utils/utilities.sh

```diff
@@ -157,7 +161,22 @@ download_generator_artifact() {
 download_protoc() {
   local protoc_version=$1
   local os_architecture=$2
-  if [ ! -d "protoc-${protoc_version}" ]; then
+
+  protoc_dirname="protoc-${protoc_version}"
+  protoc_path="${output_folder}/${protoc_dirname}/bin"
+  if [[ -f "${protoc_path}/protoc" ]]; then
+    return
+  fi
+
+  if [[ "${DOCKER_PROTOC_VERSION}" == "${protoc_version}" ]]; then
+    # if the specified protoc_version matches the one baked in the docker
+    # container, we just copy it into the output folder
+    mkdir -p "${output_folder}/${protoc_dirname}"
+    cp -r "${DOCKER_PROTOC_LOCATION}/${protoc_dirname}" \
```

Comment:
Can we use the protoc location specified in Dockerfile?

Reply:
Yes. We now explicitly `export` the `protoc_path` variable to simply point to this location instead of copying it.